Makefile

```diff
@@ -272,6 +272,20 @@ install-govmomi:
 
 test: install-govmomi portlayerapi $(TEST_JOBS)
 
+push:
+	./infra/scripts/replace-running-components.sh
```

Comment:
Use `$(BASE_DIR)//infra/scripts/replace-running-components.sh` - I know it's not consistent throughout the Makefile currently but it should be.